k8s/deployment-init-container.yaml

```diff
@@ -71,19 +71,22 @@ spec:
 
           livenessProbe:
             httpGet:
-              path: /
-              port: 10000
-            initialDelaySeconds: 15
+              path: /ready
+              port: 9901
+            initialDelaySeconds: 10
             periodSeconds: 5
 
           readinessProbe:
             httpGet:
-              path: /
-              port: 10000
-            initialDelaySeconds: 15
+              path: /ready
+              port: 9901
+            initialDelaySeconds: 10
```

Comment:
I'm not sure that using a `readinessProbe` together with a `livenessProbe` on the same endpoint makes much sense. The former makes sure that requests aren't routed to the pod (temporarily), while the latter kills the pod outright. No need to divert traffic if the pod is killed anyway... After reading [this](https://stackoverflow.com/a/65858310) I would maybe go with a `livenessProbe` and a [`startupProbe`](https://kubernetes.io/docs/tasks/configure-pod-container/configure-liveness-readiness-startup-probes/#define-startup-probes)?